docker/test/integration/features/environment.py

```diff
@@ -47,6 +47,13 @@ def before_scenario(context, scenario):
 
     logging.info("Integration test setup at {time:%H:%M:%S.%f}".format(time=datetime.datetime.now()))
     context.test = MiNiFi_integration_test(context=context, feature_id=context.feature_id)
+
+    if "USE_NIFI_PYTHON_PROCESSORS" in scenario.effective_tags:
+        if not context.image_store.is_conda_available_in_minifi_image() and context.image_store.get_minifi_image_python_version() < (3, 8, 1):
+            scenario.skip("NiFi Python processor tests use langchain library therefore Python 3.8.1 or later is needed in the MiNiFi container.")
```

Comment:
I'd rephrase this.
```suggestion
            scenario.skip("NiFi Python processor tests use langchain library which requires Python 3.8.1 or later.")
```

Reply:
Updated in e8757f9c370b2ec92866a3731af1e6d9ae0edb74